Detections/DeviceProcessEvents/SolardWinds_SUNBURST_Process-IOCs.yaml

```diff
@@ -0,0 +1,51 @@
+id: 4a3073ac-7383-48a9-90a8-eb6716183a54
+name: SUNBURST suspicious SolarWinds child processes
+description: |
+  Identifies suspicious child processes of SolarWinds.Orion.Core.BusinessLayer.dll that may be evidence of the SUNBURST backdoor
+  References:
+  - https://www.fireeye.com/blog/threat-research/2020/12/evasive-attacker-leverages-solarwinds-supply-chain-compromises-with-sunburst-backdoor.html
+  - https://gist.github.com/olafhartong/71ffdd4cab4b6acd5cbcd1a0691ff82f
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Microsoft 365 Defender
+    dataTypes:
+      - DeviceProcessEvents
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - Persistence
+relevantTechniques:
+  - T1195.002
+query:  |
+
+  DeviceProcessEvents
```

Comment:
Recommend you pass a list and use has_any, it is more efficient on larger data sets -

     let excludeProcs = dynamic([@"\SolarWinds\Orion\APM\APMServiceControl.exe", @"\SolarWinds\Orion\ExportToPDFCmd.Exe", @"\SolarWinds.Credentials\SolarWinds.Credentials.Orion.WebApi.exe", @"\SolarWinds\Orion\Topology\SolarWinds.Orion.Topology.Calculator.exe", @"\SolarWinds\Orion\Database-Maint.exe", @"\SolarWinds.Orion.ApiPoller.Service\SolarWinds.Orion.ApiPoller.Service.exe", @"\Windows\SysWOW64\WerFault.exe"]);

And

| where not(FolderPath has_any (excludeProcs))

Reply:
Good catch, will also address this in the new PR.

BTW @shainw, are the JSON schemas in (and not already in) `.script/tests/KqlvalidationsTests/CustomTables` publicly accessible somewhere? We've previously been running `| getschema` on tables we have access to.